app/src/pages/project/SpanFilterConditionField.tsx

```diff
@@ -168,7 +174,13 @@ function filterConditionCompletions(context: CompletionContext) {
       {
         label: "Hallucinations",
         type: "text",
-        apply: "evals['Hallucination'].label == 'hallucinated'",
+        apply: "annotations['Hallucination'].label == 'hallucinated'",
```

Comment:
not sure if we are set on slowly migratin away from evals, if so this seems like a reasonable place to do it

Reply:
makes sense - I'm still not fully sure how we reconcile the wording - as in there is feedback and then there is annotations and evals under that. Hard to tell. This macro might ultimately be confusing. We might have to invest in the typehadead a bit more to make the filter bar really sing.